build/Jenkinsfile.pr

```diff
@@ -293,9 +293,16 @@ pipeline {
             dir('src/github.com/mattermost/mattermost-server/build') {
                 ansiColor('xterm') {
                     sh """
+                        # capture docker logs
+                        docker ps -aq | xargs -I % sh -c 'docker ps -a --no-trunc -f id=% > %_docker_log; echo >> %_docker_log; docker logs --details -t % >> %_docker_log'
+                        docker ps -a --no-trunc > docker_ps
+                        docker stats -a --no-stream > docker_stats
+                        zip -qr docker_logs.zip *_docker_log docker_ps docker_stats
```

Comment:
this looks great and improves the debugging when needed.
however, as is this is collecting all the docker logs from the machine and you will need to filter out those that are for the build.
if possible can you get the logs only for running docker for the build?
maybe use  `docker-compose logs <<name of the service>>`

Reply:
Good call. Updated.

Reply:
cool
that makes sense to you?

Reply:
yes. already updated the pr to log containers for the current build run